src/app/modules/home/home.router.ts

```diff
@@ -20,4 +20,4 @@ const homeRoutes: Routes = [
     RouterModule
   ]
 })
-export class HomeRouterModule {}
+export class HomeRouterModule {}
```

Comment:
This file shouldn't be modified. You've removed a newline at the end. Please add it back.